CLAUDE.md

```
@@ -0,0 +1,153 @@
+J. Fluid Mech., 1010, A2 (2025) [32 pages];\\
+(OA) DOI: \href{https://doi.org/10.1017/jfm.2025.237}{10.1017/jfm.2025.237}.
+```
+
+Key formatting rules:
+- Use `\\` at end of each line except the last
+- Page count always in format `[X pages]` 
+- Semicolon `;` before final `\\` on journal line
+- For non-OA papers with preprints: `DOI: \href{...}{...}, OA: \href{...}{...}`
+- Special notes (Cover, Editor's Suggestion) go on a new line or as sub-items
+
+### For Vatsal_CV-shortCV.tex (Short CV)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)